frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -210,12 +257,19 @@ const ClusterServiceVersionStatus: React.FC<ClusterServiceVersionStatusProps> =
 export const ClusterServiceVersionTableRow = withFallback<ClusterServiceVersionTableRowProps>(
   ({ obj, rowKey, subscription, catalogSourceMissing, index, style }) => {
     const { displayName, provider, version } = _.get(obj, 'spec');
+    const ns = UIActions.getActiveNamespace();
```

Comment:
Don't get the active namespace this way. Connect to Redux.

Reply:
Does `UIActions.getActiveNamespace()` have a correct use now, or is it tech debt?